<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2025 Dice Battle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt 'Segoe UI'; color: #FFFFFF; background-color: #3a3a3a; padding: 10px; border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>·Éõ·Éù·Éó·Éê·Éõ·Éê·É®·Éî 1-·Éò·É° ·ÉØ·Éî·É†·Éò·Éê</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="playersLayout">
      <item>
       <layout class="QVBoxLayout" name="player1Layout">
        <item>
         <widget class="QLabel" name="player1Label">
          <property name="styleSheet">
           <string notr="true">font: bold 14pt; color: #00aaff;</string>
          </property>
          <property name="text">
           <string>·Éõ·Éù·Éó·Éê·Éõ·Éê·É®·Éî 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="diceImage1">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="palette">
           <palette>
            <active/>
            <inactive/>
            <disabled/>
           </palette>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="scorePlayer1">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Franklin Gothic Medium</family>
            <pointsize>24</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 24pt; color: black;</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="player2Layout">
        <item>
         <widget class="QLabel" name="player2Label">
          <property name="styleSheet">
           <string notr="true">font: bold 14pt; color: #ff55dd;</string>
          </property>
          <property name="text">
           <string>·Éõ·Éù·Éó·Éê·Éõ·Éê·É®·Éî 2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="diceImage2">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="scorePlayer2">
          <property name="font">
           <font>
            <family>Franklin Gothic Medium</family>
            <pointsize>24</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 24pt; color: black;</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="rollButton">
        <property name="styleSheet">
         <string notr="true">font: bold 14pt; background-color: #0077cc; color: white; padding: 10px 20px; border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>üé≤ ·Éí·Éê·Éí·Éù·É†·Éî·Éë·Éê</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="styleSheet">
         <string notr="true">font: bold 14pt; background-color: #444444; color: white; padding: 10px 20px; border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>üîÑ ·Éí·Éê·Éú·É£·Éö·Éî·Éë·Éê</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
